charged-up-2023/subteams/software/photon-vision.md

```diff
@@ -1,18 +1,52 @@
 # PhotonVision
 
-## What is PhotonVision
+## What is PhotonVision?
 
 The goal of PhotonVision is to get the limelight to sense the AprilTags. PhotonVision is the only compatible software we found online that works with FRC. PhotonVision is essenstially OS that runs on the limelight. For more information visit their main website: [PhotonVision Main Site](https://docs.photonvision.org/en/latest/)
 
 ## How to Set Up PhotonVision
```

Comment:
Do we have anything for this section? If not, let's add a todo with someone's name attached to it. For example: `TODO(Matt): Write this section`